src/common/Card.js

```diff
@@ -6,10 +6,12 @@ export default function Card({ children }) {
 }
 
 const CardBG = styled.div`
+  display: inline-block;
   background: white;
   border-radius: 12px;
   box-shadow: var(--primary-shadow);
   margin: 30px 0;
   padding: 15px;
   position: relative;
+  height: auto;
```

Comment:
`height: auto;` means that the element height will depend on the children's height, which is already the case. So you don't need it here.